### ISSUE TITLE
Rolls-over non-checklist items

### ISSUE THREAD

**Issue by taylorjason** [Newbrict/obsidian-obligator#33]
Not sure if this is a bug or a feature. Obligator seems to rollover ALL items from previous notes. In my testing it rolls over EVERY heading and list from previous notes. I use daily notes to capture lists of things for historical purposes and don't want all items captured in rollover, only unchecked items.  Is this a bug or feature?

**Comment by Newbrict** [comment]
Can you show a screenshot example of what you mean? Everything gets rolled over *except* for checked items.

**Comment by taylorjason** [comment]
![CleanShot 2024-07-22 at 22 15 13@2x](https://github.com/user-attachments/assets/2ebd1380-5c48-4e0b-a074-7a39c25f8857)
![CleanShot 2024-07-22 at 22 15 08@2x](https://github.com/user-attachments/assets/537b8954-750f-4dc3-b2c5-4b8b722b3660)

**Comment by taylorjason** [comment]
The grateful for section gets copied also.  If I limit the tasks to only certain headings then I can use the initial/terminal headings but that forces my tasks to key headings. Is that just the way the plugin works?

**Comment by Newbrict** [comment]
Obligator only deletes checked boxes by default, everything else is carried over. Using the initial & terminal headings give you a way to narrow the copy scope of obligator. So in short, yeah that's just how it works.

I'm confused on the desired behavior though. You want only check boxes to be copied over, no headings or text fields?

**Comment by taylorjason** [comment]
Correct, so the way I use daily notes I have tasks and various lines of text (external links, links to other notes, general text for a thought, etc). I only want the tasks to be copied over.  Headings yes, but only lines that are tasks. Nothing else.

Ideally it would be an option in the settings. "Move all items as opposed to move only unchecked tasks".

**Comment by Newbrict** [comment]
So can I treat this as a feature request for:

"Option to ignore all non-heading, non-checkbox items when making a new note"?
